cluster-state-service/handler/regex/validate.go

```diff
@@ -50,3 +53,13 @@ func IsInstanceARN(instanceARN string) bool {
 	}
 	return false
 }
+
+// IsEntityVersion validates an entity version as a positive integer
+func IsEntityVersion(entityVersion string) bool {
+	if value, err := strconv.ParseInt(entityVersion, 10, 64); err == nil {
```

Comment:
We should probably at least log this error here. We might lose info that might be useful since we're not returning the error.